test_project.py

```
@@ -18,6 +18,236 @@ def test_function_n():
+
+
+def test_metadata_extractor_on_periodontology_tei():
+    # Verify metadata extracted from a real TEI file in data/xml
+    tei_path = (
+        "Projects Python CS50 Harvard University/final_project/PaperSlicer/data/xml/"
+        "Periodontology 2000 - 2023 - Liñares - Critical review on bone grafting during immediate implant placement.tei.xml"
+    )
+    from paperslicer.extractors.metadata import TEIMetadataExtractor
+
+    md = TEIMetadataExtractor().from_file(tei_path)
+    assert md["title"] == "Critical review on bone grafting during immediate implant placement"
```

Ruff (0.12.2)

Use of `assert` detected

(S101)